README.md

```diff
@@ -11,7 +11,7 @@ For 1.x, go [here](https://github.com/twilio/twilio-video.js/tree/support-1.x/).
 
 twilio-video.js allows you to add real-time voice and video to your web apps.
 
-* [API Docs](//media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta8/docs)
+* [API Docs](//media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta9/docs)
```

Comment:
you may already be aware, but looks like beta9 link is not working yet.

Reply:
@makarandp0 Yes. I'll only merge the PR after publishing the sdk release and docs.